packages/components/src/FormatFile/FormatFileDeleteButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { Button } from "../Button";
+import { ConfirmationModal } from "../ConfirmationModal";
+
+interface DeleteButtonProps {
+  size?: "base" | "large";
+  onDelete?: () => void;
+}
+
+export function FormatFileDeleteButton({ size, onDelete }: DeleteButtonProps) {
+  const buttonSize = size === "base" ? "small" : "base";
+  const [deleteConfirmationOpen, setDeleteConfirmationOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        onClick={() => setDeleteConfirmationOpen(true)}
+        variation="destructive"
+        type="tertiary"
+        icon="trash"
+        ariaLabel="Delete Thumbnail"
```

Comment:
Not sure if "Delete thumbnail" is quite the right term here. Delete file? Or the name of the file. The confirm modal says "are you sure you want to delete this **file**"